browser-interface/packages/entryPoints/startPreview.ts

```diff
@@ -4,17 +4,18 @@ import type { IUnityInterface } from 'unity-interface/IUnityInterface'
 import { DEBUG_WS_MESSAGES } from 'config/index'
 import { getPreviewSceneId, loadPreviewScene, reloadPlaygroundScene } from 'unity-interface/dcl'
 import { logger } from './logger'
+import { kernelConfigForRenderer } from 'unity-interface/kernelConfigForRenderer'
 
 export async function startPreview(unityInterface: IUnityInterface) {
   getPreviewSceneId()
     .then((sceneData) => {
       if (sceneData.sceneId) {
-        unityInterface.SetKernelConfiguration({
-          debugConfig: {
-            sceneDebugPanelTargetSceneId: sceneData.sceneId,
-            sceneLimitsWarningSceneId: sceneData.sceneId
-          }
-        })
+        let kernelConfig = kernelConfigForRenderer()
+        kernelConfig.debugConfig = {
+          sceneDebugPanelTargetSceneId: sceneData.sceneId,
+          sceneLimitsWarningSceneId: sceneData.sceneId
+        }
+        unityInterface.SetKernelConfiguration(kernelConfig)
```

Comment:
What is the reason behind all those `SetKernelConfiguration` changes?
I don't think those changes are ok
`SetKernelConfiguration` works like a `Partial` meaning that only updated properties are sent and updated on the renderer
But these changes are constantly defaulting the config and always sending the whole object, overwriting the previously set properties on the renderer side

Reply:
Oh sorry, I didn't know that we needed only the partial information there. I will change it keeping the partials parts as it was but adding the new `urlParamsForWearablesDebug` field that I need for this feature 👍 Thanks for the feedback!

Reply:
that part is OK, adding that property in `kernelConfigForRenderer` will sent it on the first call of `SetKernelConfiguration` here https://github.com/decentraland/unity-renderer/blob/dev/browser-interface/packages/unity-interface/dcl.ts#L61

Reply:
Done here: https://github.com/decentraland/unity-renderer/pull/4292/commits/c2b12ea429e017451e6fcf18c774162d6885a664 Thanks!